core/src/templates/components/local-footer/local-footer.twig

```diff
@@ -111,6 +111,7 @@
         Use the `cell1` variable instead.
       #}
       {% if address|render|striptags|trim is not empty %}
+      <h2 class="su-sr-only-element">Address</h2>
```

Comment:
I'm adding this to make it easier for screen readers to find the address.

Reply:
This works for me.